injector/src/test/java/edu/ucr/cs/riple/injector/ClassSearchTest.java

```diff
@@ -56,8 +56,7 @@ public void annotation_declaration() {
             "package com.test;",
             "import javax.annotation.Nullable;",
             "public @interface Main{",
-            "   @Nullable",
-            "   public String foo();",
+            "   @Nullable public String foo();",
```

Comment:
Same here, most developers would expect this to be on a line above the method declaration.